### DISCUSSION TITLE
Reset Super Admin infos

### DISCUSSION THREAD

**Discussion by matheo-lucak** [strapi/strapi#7846] [Legacy Topics]
Hello,

Since Strapi migration from 3.0.x to 3.1.x, the "http://localhost:1337/admin/" page asks for email and password instead of username and password for login.
Is there a way to get or reset Super Admin infos from server CLI or to connect again with Username ?

**Comment by derrickmehaffy** [comment]
You can manually edit the database to change the username or email (not the password) or just delete the admin from the database and if that was your only one, it will redirect you to register a new one again.

**Reply by derrickmehaffy** [reply]
(You will no longer be allowed to login with username on the admin panel, this was due to us changing to passport for authentication in the admin)
